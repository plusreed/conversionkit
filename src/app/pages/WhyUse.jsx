import React from 'react';
import faqs from '../data/faq';

const WhyUse = () => (
    <div className="p-4">
        {faqs.map(faq => (
            <div key={faq.id}>
                <h1 className="font-bold text-2xl">{faq.question}</h1>
                <p className="font-normal text-md">{faq.answer}</p>
            </div>
        ))}
    </div>
);

export default WhyUse;