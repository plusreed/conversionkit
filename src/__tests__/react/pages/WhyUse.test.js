import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer'
import WhyUse from '../../../app/pages/WhyUse';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<WhyUse />, div);
  ReactDOM.unmountComponentAtNode(div);
});


it('matches snapshot', () => {
  const tree = renderer
    .create(<WhyUse />)
    .toJSON()

  expect(tree).toMatchSnapshot()
});
