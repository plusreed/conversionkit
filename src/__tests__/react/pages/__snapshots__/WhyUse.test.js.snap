// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="p-4"
>
  <div>
    <h1
      className="font-bold text-2xl"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Why should I use ConversionKit?",
        }
      }
    />
    <p
      className="font-normal text-md"
      dangerouslySetInnerHTML={
        Object {
          "__html": "ConversionKit is free and open source, works offline, and requires no account to use. We also don't track what you do.",
        }
      }
    />
  </div>
  <div>
    <h1
      className="font-bold text-2xl"
      dangerouslySetInnerHTML={
        Object {
          "__html": "How does this website work offline?",
        }
      }
    />
    <p
      className="font-normal text-md"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Magic. (In all seriousness, it's just a service worker. Everything is done in your browser and isn't sent off to a server.)",
        }
      }
    />
  </div>
  <div>
    <h1
      className="font-bold text-2xl"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Is ConversionKit open source?",
        }
      }
    />
    <p
      className="font-normal text-md"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Yes! Please visit <a href=\\"https://github.com/plusreed/conversionkit\\">https://github.com/plusreed/conversionkit</a> to see the source code.",
        }
      }
    />
  </div>
</div>
`;
